c/eca-programs/License.txt

```diff
@@ -0,0 +1 @@
+../LICENSE.Apache-2.0.txt

```

Comment:
Where does this license come from?
Are you an author or those tasks or did you ask them?
Are the programs taken from RERS2018?